File: recovered/src/com/lumiyaviewer/lumiya/render/HeadTransformCompat.java
package com.lumiyaviewer.lumiya.render;

/* loaded from: classes.dex */
public class HeadTransformCompat {
    public float lastYaw;
    public float neutralYaw;
    public float pitchDegrees;
    public float useButtonsYaw;
    public float yawDegrees;
    public final float[] rotationQuat = new float[4];
    public final float[] translationVector = new float[4];
    public final float[] headTransformMatrix = new float[16];
    public final float[] rightVectorRaw = new float[4];
    public final float[] rightVector = new float[4];
    public final float[] eulerAngles = new float[3];
    public boolean neutralYawValid = false;
    public float viewExtraYaw = 0.0f;
}

File: app/src/main/java/com/lumiyaviewer/lumiya/render/HeadTransformCompat.kt
package com.lumiyaviewer.lumiya.render

open class HeadTransformCompat {
    @JvmField var lastYaw = 0.0f
    @JvmField var neutralYaw = 0.0f
    @JvmField var pitchDegrees = 0.0f
    @JvmField var useButtonsYaw = 0.0f
    @JvmField var yawDegrees = 0.0f
    @JvmField val rotationQuat = FloatArray(4)
    @JvmField val translationVector = FloatArray(4)
    @JvmField val headTransformMatrix = FloatArray(16)
    @JvmField val rightVectorRaw = FloatArray(4)
    @JvmField val rightVector = FloatArray(4)
    @JvmField val eulerAngles = FloatArray(3)
    @JvmField var neutralYawValid = false
    @JvmField var viewExtraYaw = 0.0f
}
